src/IoC/ServiceModule.cs

```diff
@@ -15,6 +15,9 @@
     using Linn.Products.Proxy;
     using Linn.Products.Resources;
 
+    using ISaHoldStoriesReportService = Linn.Products.Domain.Linnapps.Reports.ISaHoldStoriesReportService;
```

Comment:
It's doing that fully qualified thing again. Maybe change the name of one of the classes?